src/components/NavBar/ChainSwitcher.tsx

```diff
@@ -18,32 +24,51 @@ import { NavDropdown } from './NavDropdown'
 const ChainRow = ({
   targetChain,
   onSelectChain,
+  toggleOpen,
 }: {
   targetChain: SupportedChainId
   onSelectChain: (targetChain: number) => void
+  toggleOpen: () => void
 }) => {
   const { chainId } = useWeb3React()
   const active = chainId === targetChain
-  const { label, logoUrl } = getChainInfo(targetChain)
+  const { helpCenterUrl, explorer, bridge, label, logoUrl } = getChainInfo(targetChain)
 
   return (
-    <Row
-      as="button"
-      background={active ? 'lightGrayOverlay' : 'none'}
-      className={`${styles.ChainSwitcherRow} ${subhead}`}
-      onClick={() => onSelectChain(targetChain)}
-    >
-      <ChainDetails>
-        <img src={logoUrl} alt={label} className={styles.Icon} />
-        {label}
-      </ChainDetails>
-      {active && <CheckMarkIcon width={20} height={20} />}
-    </Row>
+    <Column background={active ? 'lightGrayOverlay' : 'none'} borderRadius="12">
+      <Row
+        as="button"
+        background="none"
+        className={`${styles.ChainSwitcherRow} ${subhead}`}
+        onClick={() => (active ? toggleOpen() : onSelectChain(targetChain))}
+      >
+        <ChainDetails>
+          <img src={logoUrl} alt={label} className={styles.Icon} />
+          {label}
+        </ChainDetails>
+        {active && <ActiveNetworkIcon />}
+      </Row>
+      {active && (
+        <Column gap="8" paddingBottom="16">
+          <Box className={styles.Separator} />
+          {bridge && <ChainLinkOut externalLink={bridge} label={`${label} bridge`} />}
+          {explorer && <ChainLinkOut externalLink={explorer} label={`${label} scan`} />}
+          {helpCenterUrl && <ChainLinkOut externalLink={helpCenterUrl} label={`Learn more`} />}
+        </Column>
+      )}
+    </Column>
   )
 }
 
 const ChainDetails = ({ children }: { children: ReactNode }) => <Row>{children}</Row>
 
+const ChainLinkOut = ({ externalLink, label }: { externalLink: string; label: string }) => (
+  <Row as="a" href={externalLink} className={styles.ChainInfo} target={'_blank'} rel={'noopener noreferrer'}>
```

Comment:
I think at one point we had an external link component that added target/rel values automatically. Might be worth adding that logic here, or checking if it's already there.